spec/support/matchers/be_a_datetime_string.rb

```diff
@@ -0,0 +1,9 @@
+RSpec::Matchers.define :be_a_datetime_string do |expected_time|
+  match do |actual_string|
+    actual_time = Time.zone.parse(actual_string)
+    return false unless actual_time
+    return true unless expected_time
+
+    (actual_time - expected_time).abs < 1.second
```

Comment:
You can use `timecop` gem if you want. I can show you an example of usage if you want. https://github.com/travisjeffery/timecop

Reply:
Yes, I use it, but when some timestamp serialized it is become less _precise_ than one in ActiveRecord model. So when I compare one in a response and in a model I get assertion error.
Let me know if timecop solves this case.